src/Entity/PackageRepository.php

```diff
@@ -477,6 +480,11 @@ public function getSuggestCount($name): int
         return (int) $this->getEntityManager()->getConnection()->fetchOne($sql, $args);
     }
 
+    /**
+     * @param string $name
+     * @param int $offset
+     * @param int $limit
```

Comment:
you could use actual parameter types rather than phpdoc

Reply:
Yes please go wild with PHP 8 syntax in this repo 😄